### PR TITLE
Add support for disabling skinning of lock screen

### PR DESCRIPTION
**Pull request by NiceManiac**
Added a toggle to disable the skinning of the lock screen to enable support for custom images in from the plugin Minimal Lock Screen. This is a pull request for the solution i found in issue #52

### CODE REVIEW

File: Obsidian/shared.css
Comment (NiceManiac) [inline Obsidian/shared.css:1]: Removed the lock screen skinning from shared.css

File: Obsidian/theme.json
Comment (NiceManiac) [inline Obsidian/theme.json:1]: Updated theme.json with a toggle to enable and disable the skining of the locks creen. Defaults to Yes

**Comment by EMERALD0874** [comment]
I feel like this is a pretty specific fix for a situation caused by #28. I'll look into a fix for the !important abuse when I have a chance, but I would rather not implement specific solutions like this for the time being.

**Comment by NiceManiac** [comment]
updated my fork to work with the new updates in the repo, in case you change your mind about my fix.

**Comment by EMERALD0874** [comment]
This is solved by https://github.com/EMERALD0874/Steam-Deck-Themes/commit/4f9d5060e0707f2a26f6537552ddaabf55cba8cf, a patch which fixed incompatibility with many themes including Minimal Lock Screen.

**Pull request closed without merging**